deployments/boba.json

```diff
@@ -28,5 +28,6 @@
   "LiFiDEXAggregator": "0x6140b987d6B51Fd75b66C3B07733Beb5167c42fc",
   "EmergencyPauseFacet": "0x6F2baA7cd5F156CA1B132F7FF11E0fa2aD775F61",
   "RelayFacet": "0x424BDbbaEda89732443fb1B737b6Dc194a6Ddbd5",
-  "Permit2Proxy": "0x89c6340B1a1f4b25D36cd8B063D49045caF3f818"
-}
\ No newline at end of file
+  "Permit2Proxy": "0x89c6340B1a1f4b25D36cd8B063D49045caF3f818",
+  "LiFiTimelockController": "0x97A13f17bFeE56eEE8A030A8bDc5772DE6Ad0280"
```

Comment:
in deployments log file the address is: `0x5604A94A3438C3074EFFF803fab14B7244fe4E29`

Reply:
fixed, 
this is the correct and it's now in both log files:
0x97A13f17bFeE56eEE8A030A8bDc5772DE6Ad0280